### ISSUE TITLE
Commission new A3 colour laser-printer (Xerox Phaser 7100)

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#1821]
We've been given a colour A3 laser-printer by Open Culture, so we should bring it into use properly.

It works over USB if you print from the "bypass tray", but needs a few extra jobs before it's fully functional:

* [x] Add it to the wiki – https://github.com/DoESLiverpool/somebody-should/wiki/OfficePrinters
* [x] Create a label for it here – https://github.com/DoESLiverpool/somebody-should/labels
* [x] Connect it to the network
* [ ] Work out how to get it printing from the main paper tray

**Comment by zarino** [comment]
I’ve spent a while this evening trying to get the printer to print from Tray 1, with no success. The highlights, for anyone who follows me:

- The [product support page is here](https://www.support.xerox.com/en-us/product/phaser-7100) – under "Documentation" there’s both a 140 page "User Guide" and a 90 page "System Administrator Guide"
- The printer’s web interface is available at http://10.0.35.95
- I installed Driver version 5.13.2_2427 on macOS Monterey 12.6.7
- I’m able to send a document to print, A4 portrait, as long as A4 portrait paper is loaded in the Bypass Tray.
   - I _think_ this is because the printer doesn’t think suitable paper (A4 portrait) is loaded in Tray 1.
   - If A4 paper isn’t loaded in the Bypass Tray, then the printer screen says `Add paper–BypassTray / A4 portrait Plain`
- There are only two paper size options for Tray 1 – `Auto` (the printer autodetects, not sure how) or `Custom` (where you define a custom length and height in millimetres).
- I was initially confused that, despite `Settings` > `Admin Menu` > `Printer Settings…` > `During Loading…` being set to `Allow Change`, the printer didn’t prompt for a new print size on removing and replacing the tray.
- I also noticed Tray 1 was slightly extended (to accomodate A3 paper), and so reduced the length back to normal. It seemed to make no difference.
   - …except, after making this change, the printer started offering to "Change Tray 1 settings?" when the tray was removed and replaced, which is… weird? But good?
- Now, when the guides in Tray 1 are set to A4 portrait, and A4 paper is loaded in portrait, the "Change Tray 1 settings?" prompt reveals that the printer _thinks_ the loaded paper is `Auto 8.5x11" portrait`.
   - 8.5x11" is US Letter or ANSI Letter. Suspicious.
- I tried setting the Tray 1 paper size to both `297mm portrait (Y)` / `210mm landscape (X)` and `210mm portrait (Y)` / `297mm landscape (X)`, with the A4 portrait paper loaded, and when a job is sent to Tray 1, the paper becomes jammed.

Something is going wrong with the printer’s paper size auto-detection in Tray 1. And yet, if you override with a custom paper size, it jams when picking up the first sheet.

**Comment by johnmckerrell** [comment]
I had a little go with this printer the other day. I had no issues printing A4 from Tray 1 using Preview on a Mac, using macOS Ventura 13.6. I plugged into the printer and it showed up in the printer settings, it automatically used "Generic PostScript Printer" which I didn't hold out much hope for but the colour A4 prints I got looked great. They did seem a bit like they'd been through a filter, though the print was very high quality it almost looked like it had been modified to look like a painting or something like that.

**Comment by Simon-Jones7** [comment]
I read what's been written here and managed to successfully print A3. When you rearrange the paper tray, you need to go into the paper tray settings dialogue that the printer automatically brings up. For A3, you just select A3 and it works. For A4 it doesn't work.

- If you put A4 paper in the the tray portrait (as you're looking at it), the printer will only give you the option to set the tray to US letter size or custom. If you set A4 in custom, it doesn't work.
- If you put A4 paper in the tray landscape, the printer will only give you the option to set the tray as A5 or custom.

So basically printing A4 doesn't seem to be an option.

By the way, I'm on a Mac running Big Sur.
